packages/rollup-plugin/__tests__/index-test.js

```diff
@@ -61,29 +61,9 @@ describe('rollup-plugin-stylex', () => {
   it('extracts CSS and removes stylex.inject calls', async () => {
     const { css, js } = await runStylex({ fileName: 'stylex.css' });
 
-    expect(css).toMatchInlineSnapshot(`
-      "
-      @layer priority1, priority2, priority3, priority4;
-      @layer priority1{
-      @keyframes xgnty7z-B{0%{opacity:.25;}100%{opacity:1;}}
-      }
-      @layer priority2{
-      .x1oz5o6v:hover{background:red}
-      }
-      @layer priority3{
-      .xeuoslp{animation-name:xgnty7z-B}
-      .x1lliihq{display:block}
-      .x78zum5{display:flex}
-      .xt0psk2{display:inline}
-      .x1hm9lzh{margin-inline-start:10px}
-      }
-      @layer priority4{
-      .x1egiwwb{height:500px}
-      .xlrshdv{margin-top:99px}
-      .xh8yej3{width:100%}
-      .x3hqpx7{width:50%}
-      }"
-    `);
+    expect(css).toMatchInlineSnapshot(
+      `"@layer priority1{@keyframes xgnty7z-B{0%{opacity:.25}to{opacity:1}}}@layer priority2{.x1oz5o6v:hover{background:red}}@layer priority3{.xeuoslp{animation-name:xgnty7z-B}.x1lliihq{display:block}.x78zum5{display:flex}.xt0psk2{display:inline}.x1hm9lzh{margin-inline-start:10px}}@layer priority4{.x1egiwwb{height:500px}.xlrshdv{margin-top:99px}.xh8yej3{width:100%}.x3hqpx7{width:50%}}"`,
```

Comment:
Yeah this is worse than adding even more whitespace.

Reply:
Removed this in the newer commit. Back to the original format